FoodPin/FoodPin/RestaurantTableViewCell.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  RestaurantTableViwCell.swift
+//  FoodPin
+//
+//  Created by Viviana Mesaros on 11.04.2022.
+//
+
+import UIKit
+
+class RestaurantTableViewCell: UITableViewCell {
+    
+    @IBOutlet var nameLabel: UILabel!
+    @IBOutlet var locationLabel: UILabel!
+    @IBOutlet var typeLabel: UILabel!
+    @IBOutlet var thumbnailImageView: UIImageView!
```

Comment:
These should be private, like this:
@IBOutlet private var nameLabel: UILabel!
And you can create a public method, like "configureCell" to be able to modify these properties from the viewController